pkg/TIPA.py

```
@@ -164,587 +219,496 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+                    break
 
-                        if page2.status_code != 200:
-                            raise Exception("Error Code: ", page2.status_code)
+            except Exception as e:
+                if tryCounter == 1:
+                    URL = f"https://tarkov-market.com/item/{corrected_text}".lower().capitalize()
 
-                        else:
-                            print("Request Gamepedia 200")
+                elif tryCounter == 2:
+                    words_list = corrected_text.lower().split("_")
```

Ruff (0.8.0)

Undefined name `corrected_text`

(F821)